### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fentontaylor**
### Summary
- Adds PUT and DELETE endpoints for playlists to readme. 
- Edits POST favorites description to match updated functionality of that endpoint.

### CODE REVIEW

**Comment by fentontaylor** [comment]
@nathangthomas You can pull down this branch and add the POST playlists description before we merge this.

File: README.md [hunk 1]
```diff
@@ -68,45 +68,41 @@ https://looney-tunes.herokuapp.com
...
 ```
-**Success Response**
+
+#### Success Response
+
 ```
 Status: 201
 
 {
   "id": 5
```
Comment (nathangthomas) [inline README.md]: I double checked this in Postman and our success response is currently: 

```{
    "title": "We will Rock You",
    "artist_name": "Queen",
    "genre": "Rock",
    "rating": 88
}```

Our user story calls for a response of: 
```
Status: 201
{
  "id": 5
}
```

I'm not sure how that slipped through the cracks. Do you mind changing this before I make my changes and pull request?

Comment (fentontaylor) [reply README.md]: [Issue #14](https://github.com/fentontaylor/play/issues/14) amended the endpoint. I updated the readme to reflect the requirements of that user story.

**Comment by fentontaylor** [comment]
I finished updating the readme. I added the section for POST /api/v1/playlists as well

File: README.md [hunk 2]
```diff
@@ -29,96 +32,108 @@ knex migrate:latest
...
-**Success Response**
+
+#### Success Response
+
 ```
 Status: 201
 
 {
-  "id": 5
+  "id": 5,
```
Comment (nathangthomas) [inline README.md]: I went back and double checked the assignment. This is what they were asking for the response. Thanks for updating it!

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Sweater Weather Express
```
Comment (nathangthomas) [inline README.md]: Good catch :-)

**Pull request merged**